src/requests/requestWeather.js

```diff
@@ -1,67 +1,49 @@
-const https = require('https');
-const http = require('http');
+const axios = require('axios');
```

Comment:
Atualizar imagem do docker adicionando modulo axios: 1:23  error  Unable to resolve path to module 'axios'

Reply:
Rode de novo, dê um `docker-compose build` antes.